docs/GraFx-Fonts/overview/elements/index.md

```diff
@@ -0,0 +1,43 @@
+# Application elements of GraFx Fonts
+
+When opening GraFx Fonts, you see the action banner and below all font families.
+
+![appscreen](dashboard.png)
+
+## Action banner
+
+The area of the application where you interact with the contents. You can search, sort and change the view.
+
+### Sort Field
+
+Select the field you want to sort the contents below
```

Comment:
Select the field you want to sort the contents below on

Reply:
Done